bin/csv2json

```diff
@@ -57,11 +57,16 @@ csv_reader   = csv.DictReader(csv_file)
 csv_rows         = []
 csv_row_previous = None
 
+def should_fill_cell(cell_value, previous_row, row_key):
+    cell_is_empty        = not cell_value
+    previous_row_exists  = previous_row
+    column_should_repeat = row_key in ['Feature', 'Verification Goal']
+    return  cell_is_empty  and  previous_row_exists  and  column_should_repeat
```

Comment:
This filtering doesn't handle every possible case, because the ".xlsx" files are haphazardly and inconsistently formatted.

(I have ideas for how to fix this in the future. But let's compromise for now since this is just a test run?)